packages/application-components/src/components/public-page-layout/index.ts

```diff
@@ -1 +1,8 @@
-export { default } from './public-page-layout';
+import { lazy } from 'react';
+
+const PublicPageLayout = lazy(
+  () =>
+    import('./public-page-layout' /* webpackChunkName: "public-page-layout" */)
+);
```

Comment:
This is now code splitted, mostly to avoid strange issues related to loading the backround png image (e.g. vrts, components-playground).

Reply:
Yes that's fine because the AppShell has Suspense already. Also, this specific component is only used in the authentication app so far.

Thanks for asking!

Reply:
Cool 🧊